pep257.py

```diff
@@ -100,7 +123,9 @@ class Module(Definition):
     _nest = staticmethod(lambda s: {'def': Function, 'class': Class}[s])
     module = property(lambda self: self)
     all = property(lambda self: self._all)
```

Comment:
I'm happy to change those, but the goal was to fix the test failures, not make everything more conventional.